lib/diagnostic.rb

```diff
@@ -29,31 +31,34 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = "{'#character'} is really a Sith Lord."
```

Comment:
String interpolation in Ruby is of the form: `"#{...}"`.  This should be `"#{character} is really a Sith Lord."`.